bamboo/deploy-dev-integration-test-stack.sh

```diff
@@ -21,12 +21,12 @@ if [[ $USE_CACHED_BOOTSTRAP == true ]]; then
 fi
 echo "***Deploying stack with built source"
 
+# Extract cache of compiled TS files
+./bamboo/extract-ts-build-cache.sh
+
 npm install
-## Double bootstrapping required as workaround to
-## lerna re-bootstrapping issue in older releases
-## (similiar to  https://github.com/lerna/lerna/issues/1457)
-(npm run bootstrap-no-build-no-scripts || true) && npm run bootstrap-no-build
+npm run bootstrap-no-build-no-scripts-ci
```

Comment:
See comment above re: this

Reply:
c1f000b1a427393ee1699529e6e1e362d9b06b11